### PR TITLE
ah_installed: introduce /etc/iiab/iiab_state.yml and begin breaking apart playbooks into install.yml + setup.yml + enable.yml

### PR DESCRIPTION
**Pull request by jvonau**
#2020 rebased to master

### CODE REVIEW

**Comment by holta** [comment]
Can @jvonau's improvements of PR https://github.com/holta/iiab/pull/307 + the eight 1-liner suggestions on that ticket...be added here?

**Comment by jvonau** [comment]
Think I caught them when I was rebasing but poke away..

File: iiab-install [hunk 1]
```diff
@@ -133,22 +131,61 @@ if [ -f /etc/iiab/iiab.env ]; then
...
+# changes in Stage 3 to account for post image creation, once the above is run.
+# reinstall=True would force kiwix to re-download and re-install in commit
+# ce2ec3b0cad76449caf3299003b5d297a3164181
+## End image catch up
+fi
+
+# Allow iiab-install to read IIAB_STATE_FILE to not repeat installs of previous
+# roles that already completed within the stage.
+if [ ! -f  $IIAB_STATE_FILE ]; then
+        mkdir /etc/iiab/
```
Comment (holta) [inline iiab-install]:
```suggestion
```

File: iiab-install [hunk 2]
```diff
@@ -133,22 +131,61 @@ if [ -f /etc/iiab/iiab.env ]; then
...
+    fi
```
Comment (holta) [inline iiab-install]:
```suggestion
```

**Review by holta — COMMENTED** [review]
Plz remove these 2 lines, per PR holta#307 suggestions.

**Comment by holta** [comment]
As per PR #2020 I recommend we remove or comment out https://github.com/jvonau/iiab/blob/ah_installed/runrole#L56-L77 which has a few errors in its regexp's.

IMO, the above 20+ lines should be replaced by: https://github.com/jvonau/iiab/blob/ah_installed/runrole#L25-L28

File: iiab-install (hunk 2 above)
Comment (holta) [inline iiab-install]: Oops lines 180-181 have re-appeared, can these be re-deleted hopefully?

**Comment by jvonau** [comment]
You are free to rebase and test #2020 but the rebase is a bit tricky.

**Pull request closed without merging**